src/js/utils/paste-utils.js

```diff
@@ -11,11 +10,14 @@ export function setClipboardCopyData(copyEvent, editor) {
 
   const range = cursor.offsets;
   const mobiledoc = post.cloneRange(range);
+  const fragment = getSelectionContents();
+  const div = createDiv();
+  div.appendChild(fragment);
   const html =
     `<div data-mobiledoc='${JSON.stringify(mobiledoc)}'>` +
-      `<a href='${issueUrl}'>Pasting from Content-Kit not yet supported.</a>` +
+      div.innerHTML +
```

Comment:
Using the `mobiledoc-html-renderer` or `mobiledoc-dom-renderer` here would definitely be more correct. I am reluctant to bring those in as deps for what seems like a fairly small use-case at the moment.

The use-case I'm targeting here is to make copy-pasting from a content-kit instance into something else non-surprising (as opposed to semantically accurate), so that copy/pasting out of content-kit and into google docs or medium displays something that looks right.

Because we hide a serialized mobiledoc in the clipboard data when copying, pasting from one content-kit editor instance to another is always a perfect reproduction (modulo any cards used that aren't present in the target's editor).